Jobs [Distributed Scheduler] API docs, pt.1 (#4164)
dapr/docs: daprdocs/content/en/concepts/building-blocks-concept.md

| [**Configuration**]({{< ref "configuration-api-overview.md" >}}) | `/v1.0/configuration` | The Configuration API enables you to retrieve and subscribe to application configuration items for supported configuration stores. This enables an application to retrieve specific configuration information, for example, at start up or when configuration changes are made in the store. 
| [**Distributed lock**]({{< ref "distributed-lock-api-overview.md" >}}) | `/v1.0-alpha1/lock` | The distributed lock API enables you to take a lock on a resource so that multiple instances of an application can access the resource without conflicts and provide consistency guarantees.  
| [**Cryptography**]({{< ref "cryptography-overview.md" >}}) | `/v1.0-alpha1/crypto` | The Cryptography API enables you to perform cryptographic operations, such as encrypting and decrypting messages, without exposing keys to your application.

> | [**Scheduler**]({{< ref "scheduler-overview.md" >}}) | `/v1.0-alpha1/scheduler` | The Scheduler API enables you to schedule and orchestrate jobs at scale in your distributed environment.

Reviewer: ```suggestion
| [**Scheduler**]({{< ref "scheduler-overview.md" >}}) | `/v1.0-alpha1/scheduler` | The Scheduler API enables you to schedule and orchestrate jobs. Example scenarios include schedule batch processing jobs to run at 5 PM every business day, schedule various maintenance scripts to perform clean-ups, or schedule ETL jobs to run at specific times (hourly, daily) to fetch new data, process it, and update the data warehouse with the latest information.
```

> | [**Distributed Scheduler**]({{< ref "cryptography-overview.md" >}}) | `/v1.0-alpha1/scheduler` | The Scheduler API enables you to schedule and orchestrate jobs at scale in your distributed environment.

Reviewer: ```suggestion
| [**Scheduler**]({{< ref "scheduler-overview.md" >}}) | `/v1.0-alpha1/scheduler` | The Scheduler API enables you to schedule and orchestrate jobs at scale.
```

> | [**Jobs**]({{< ref "jobs-overview.md" >}}) | `/v1.0-alpha1/jobs` | The Jobs API enables you to schedule and orchestrate jobs. Example scenarios include: <ul><li>Schedule batch processing jobs to run at 5 PM every business day</li><li>Schedule various maintenance scripts to perform clean-ups</li><li>Schedule ETL jobs to run at specific times (hourly, daily) to fetch new data, process it, and update the data warehouse with the latest information.</li></ul>

Reviewer: ```suggestion
| [**Jobs**]({{< ref "jobs-overview.md" >}}) | `/v1.0-alpha1/jobs` | The Jobs API enables you to schedule and orchestrate jobs. Example scenarios include: <ul><li>Schedule batch processing jobs to run every business day</li><li>Schedule various maintenance scripts to perform clean-ups</li><li>Schedule ETL jobs to run at specific times (hourly, daily) to fetch new data, process it, and update the data warehouse with the latest information.</li></ul>
```
